src/modules/roc_netio/target_libuv/roc_netio/udp_sender_port.cpp

```diff
@@ -11,10 +11,17 @@
 #include "roc_core/helpers.h"
 #include "roc_core/log.h"
 #include "roc_core/panic.h"
+#include "roc_netio/sendto.h"
 
 namespace roc {
 namespace netio {
 
+namespace {
+
+const core::nanoseconds_t PacketLogInterval = 5 * core::Second;
```

Comment:
Just tested, the optimization works as expected. Though 5 seconds seems to be too frequent, could you please change it to 20 seconds?